tests/unit/test_hnsw.cpp

```diff
@@ -1655,140 +1655,6 @@ TYPED_TEST(HNSWTest, testInitialSizeEstimation_No_InitialCapacity) {
     VecSimIndex_Free(index);
 }
 
-TYPED_TEST(HNSWTest, testIncomingEdgesSize) {
```

Comment:
This test is no longer needed as we now have no guarantee on the behavior of choosing between candidates with equal distance (aka choosing at random).